### PR TITLE
feat: search functional and mobile responsive

### PR DESCRIPTION
**Pull request by PuneetP16**
In this PR I have worked on the responsiveness and made search work dynamic on key press.

Made search box mobile responsive

image

### CODE REVIEW

**Review by m26unkwn — APPROVED** [review]
Everything seems good to me.

The code is readable and clean.

File: src/components/UI/Aside/Aside.css [hunk 1]
```diff
@@ -36,14 +36,18 @@
...
+    display: flex;
+    flex-flow: row nowrap;
+    list-style-type: none;
+    justify-content: space-evenly;
+    align-items: center;
+    width: 100%;
+    position: fixed;
+    bottom: 4.6rem;
+    padding: 1.2rem 0;
+    background: var(--COLOR-BG);
```
Comment (dev-ashishsethi) [inline src/components/UI/Aside/Aside.css:49]: give fallback values in when using variables

**Review by dev-ashishsethi — APPROVED** [review]
good to merge

File: src/components/UI/Aside/Aside.css (hunk 1 above)
Comment (PuneetP16) [reply src/components/UI/Aside/Aside.css:49]: will look into it.

**Pull request merged**